### DISCUSSION TITLE
how to play hls if using player.html

### DISCUSSION THREAD

**Discussion by nocpbxchap** [ant-media/Ant-Media-Server#6422] [General]
hello, i want to play streams with HlS,  but currently only able to play hls on play.html and i need to work with player.html as we user using these videoplayer.

**Comment by amarantmeida** [comment]
Hello @nocpbxchap 
We have a detailed documentation on how you can use the [Ant Media Server Web Player](https://antmedia.io/docs/guides/playing-live-stream/embedded-web-player/#ant-media-server-web-player). Please check and let us know if that helps you.

**Reply by nocpbxchap** [reply]
seems i have to change player, thanks.

**Reply by Mohit-3196** [reply]
Hi @nocpbxchap,
How are you?

The sample player, player.html is a dedicated webrtc sample player and therefore you cannot play HLS with it.
To play HLS, you can use the embedded player, play.html with &playOrder=hls

Thank you.
